src/helm/benchmark/scenarios/unitxt_scenario.py

```diff
@@ -0,0 +1,66 @@
+from typing import List
+from datasets import load_dataset
+
+from helm.benchmark.scenarios.scenario import (
+    Output,
+    Reference,
+    Scenario,
+    Instance,
+    Input,
+    CORRECT_TAG,
+    TRAIN_SPLIT,
+    TEST_SPLIT,
+    VALID_SPLIT,
+)
+
+
+class UnitxtScenario(Scenario):
+    """Integration with https://www.unitxt.org/"""
```

Comment:
```suggestion
    """Integration with https://unitxt.rtfd.io/"""
```
The `unitxt.org` address is under approval process and shut down for the near time